### PR TITLE
fix boilerplate

### PR DESCRIPTION
**Pull request by islinwb**
as part of fix for https://github.com/istio/istio/issues/8996

### CODE REVIEW

File: istioctl/cmd/istioctl/convert_ingress.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2017 Istio Authors.
+// Copyright 2018 Istio Authors
 //
```
Comment (stewart-yu) [inline istioctl/cmd/istioctl/convert_ingress.go]: IMO， we should not change the year. but also seems ok

Comment (fisherxu) [reply istioctl/cmd/istioctl/convert_ingress.go]: year should be the file created, not modified :)

**Pull request merged**